packages/ui/src/council/components/election/announcing/AnnounceCandidateButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useCallback } from 'react'
+
+import { ButtonPrimary } from '@/common/components/buttons'
+import { useModal } from '@/common/hooks/useModal'
+import { AnnounceCandidateModalCall } from '@/council/modals/AnnounceCandidate'
+
+export const AnnounceCandidateButton = () => {
```

Comment:
Let's keep that in [sync with the handbook](https://joystream.gitbook.io/joystream-handbook/governance/council#announce-candidacy).